client/src/modules/cohort/index.jsx

```diff
@@ -172,6 +196,7 @@ class Cohort extends PureComponent {
     return (
       <Fragment>
         <Toolbar />
```

Comment:
I think that the toolbar should be moved to the Layout component now. All components render the toolbar but don't use it in any way.